### PR TITLE
Parry Code Improvement

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
I was dumber when I coded this and it became base for a lot of people, I'm FIXING IT.

## Why It's Good For The Game
It's literally just a code improvement. Uses existing tools instead of trying to do everything manually.

### CODE REVIEW

**Comment by Coxswain-Navigator** [comment]
It's an improvement.

**Review by monochromatism — APPROVED** [review]
Significant improvement in code quality, no functionality lost.

File: code/game/objects/items/ego_weapons/_ego_weapon.dm [hunk 1]
```diff
@@ -176,44 +190,26 @@
 /obj/item/ego_weapon/shield/proc/AnnounceBlock(mob/living/carbon/human/source, damage, damagetype, def_zone)
 	SIGNAL_HANDLER
 	block_success = TRUE
-	var/src_message = "<span class='userdanger'>[source.real_name] [hit_message]</span>"
 
 	playsound(get_turf(src), block_sound, 50, 0, 7)
-	for(var/mob/living/carbon/human/person in view(7, source))
-		to_chat(person, src_message)
+	source.visible_message("<span class='userdanger'>[source.real_name] [hit_message]</span>")
```
Comment (Kitsunemitsu) [inline code/game/objects/items/ego_weapons/_ego_weapon.dm:195]: lmao

**Review by Kitsunemitsu — APPROVED** [review]
Cleanup and clearer variable names

**Pull request merged**